etc/input.conf

```diff
@@ -57,17 +58,17 @@
 #Ctrl+RIGHT  no-osd sub-seek  1         # seek to the next subtitle
 #Ctrl+Shift+LEFT sub-step -1            # change subtitle timing such that the previous subtitle is displayed
 #Ctrl+Shift+RIGHT sub-step 1            # change subtitle timing such that the next subtitle is displayed
-#Alt+left  add video-pan-x  0.1         # move the video right
-#Alt+right add video-pan-x -0.1         # move the video left
-#Alt+up    add video-pan-y  0.1         # move the video down
-#Alt+down  add video-pan-y -0.1         # move the video up
+#Alt+LEFT  script-binding positioning/pan-x -0.2 # pan to the right
+#Alt+RIGHT script-binding positioning/pan-x  0.2 # pan to the left
+#Alt+UP    script-binding positioning/pan-y -0.2 # pan down
+#Alt+DOWN  script-binding positioning/pan-y  0.2 # pan up
```

Comment:
Does it need to be changed? Could we use `Ctrl+Alt` modifier for new behavior? Also could update numpad bindings.

Reply:
No idea, up to you if you want to keep video-pan, I never used it and never understood why you would want to move a video beyond the visible window. But it's probably not worth binding both especially since ctrl+alt+arrows is inconvenient to press. I would like to have different default key bindings for images where they can just be bound to arrows but I'm not sure how that could be done if we don't want to use input sections.

Reply:
> ctrl+alt+arrows is inconvenient to press

I don't see any difference to `alt+arrows`. Can't do that realistically with one hand either.

> I would like to have different default key bindings for images where they can just be bound to arrows but I'm not sure how that could be done if we don't want to use input sections.

I was thinking some time ago about this, because lots of key bindings could be different based on image/video context. This is different topic tho.

> Also if scripts are disabled these commands now do nothing which isn't good.

This is exactly what I worry too. mpv should be usable without lua backed compiled in, we shouldn't replace core features with script counterpart for important key bindings. Not sure if `video-pan-` is important, but I would leave it as-is to avoid this issue. For first-party input config we should prefer native / always available features.